django/thunderstore/repository/migrations/0024_comment.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.1.4 on 2021-01-03 16:27
+
+import uuid
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("repository", "0023_add_cache_refresh_celery_schedule"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Comment",
+            fields=[
+                ("object_id", models.PositiveIntegerField()),
+                ("content", models.TextField(max_length=2048)),
+                ("date_created", models.DateTimeField(auto_now_add=True)),
+                ("date_updated", models.DateTimeField(auto_now=True)),
+                (
+                    "uuid4",
+                    models.UUIDField(
+                        default=uuid.uuid4,
```

Comment:
Default isn't calling ulid, although this migration is going to be regenerated by the looks of things, which might fix the situation